packages/manager/modules/manager-pci-common/src/translations/index.ts

```diff
@@ -0,0 +1,9 @@
+export * from './announcement-banner';
+export * from './bytes';
+export * from './common';
+export * from './discovery-banner';
+export * from './free-local-zones-banner';
+export * from './trusted-zone-banner';
+export * from './flavor-selector';
+export * from './quantity-selector';
+export * from './region-selector';
```

Comment:
i think each component should import directly it's translations

if you export all and import it you will import all the translations when manager-pci-common is imported, even if some translations are not used